### PR TITLE
pythonPackages.django-logentry-admin: init at 1.0.4

### PR DESCRIPTION
**Pull request by MrMebelMan**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by mkaito — APPROVED** [review]

File: pkgs/development/python-modules/django-logentry-admin/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1xpqfr02n365vvriamz5a0a90ax2cnvfa1nkl6ai2gawdcaglcq0";
+  };
+
+  # too complicated to setup, requires a running django instance
+  doCheck = false;
+
+  # checkInputs = [ tox ];
```
Comment (jonringer) [inline pkgs/development/python-modules/django-logentry-admin/default.nix]: i would just remove this if it's relevant.
```suggestion
```

File: pkgs/development/python-modules/django-logentry-admin/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  pname = "django-logentry-admin";
+  version = "1.0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1xpqfr02n365vvriamz5a0a90ax2cnvfa1nkl6ai2gawdcaglcq0";
+  };
+
+  # too complicated to setup, requires a running django instance
+  doCheck = false;
```
Comment (risicle) [inline pkgs/development/python-modules/django-logentry-admin/default.nix]:
```
  src = fetchFromGitHub {
    owner = "yprez";
    repo = pname;
    rev = "v${version}";
    sha256 = "1ji04qklzhjb7fx6644vzikjb2196rxyi8hrwf2knsz41ndvq1l9";
  };

  checkInputs = [ pytest pytest-django ];
  checkPhase = ''
    rm -r logentry_admin __init__.py
    pytest
  '';
```

Comment (MrMebelMan) [reply pkgs/development/python-modules/django-logentry-admin/default.nix]: oh... wow :)

**Comment by mmahut** [comment]
@GrahamcOfBorg build pythonPackages.django-logentry-admin python3Packages.django-logentry-admin

**Pull request merged**